skema/text_reading/text_reading/grounding/src/test/scala/org/ml4ai/grounding/HyperParameterSearch.scala

```diff
@@ -0,0 +1,87 @@
+package org.ml4ai.grounding
+
+import breeze.linalg.DenseVector
+import breeze.stats.distributions.{Gaussian, Uniform}
+import com.typesafe.config.ConfigFactory
+import org.clulab.utils.FileUtils
+
+import scala.collection.immutable.ListMap
+
+/** *
+  * HyperParameterSearch : generic, bruteforce Hyper parameter search using breeze (numpy style) calculations.
+  * Follows:
+  * For each concept.name
+  * Find edit distance between text to be grounded and the concept.name
+  * normalized edit distances = Edit distance/length of longest string among (text to be grounded, concept.name)
+  * alpha * cosineSimilarities + (1-alpha) (1-normalized edit distances)
+  */
+object HyperParameterSearch extends App {
+
+  /** *
+    * Get accuracy for this lambda, alpha values provided.
+    * MiraEmbeddingsGrounder is initialized for lambda, alpha value.
+    * @param lambda
+    * @param alpha
+    * @return accuracy
+    */
+  def getAccuracyForThisHyperParams(lambda: Float, alpha: Float, path: String): Float = {
+    val miraEmbeddingsGrounderGS: MiraEmbeddingsGrounder = {
+      val config = ConfigFactory.load().getConfig("Grounding")
+      val ontologyPath = config.getString("ontologyPath")
+      // val embeddingsPath = config.getString("embeddingsPath")
+      MiraEmbeddingsGrounder(ontologyPath, None, lambda, alpha)
+    }
+
+    val targets = {
+      // Drop the first line that is the header
+      FileUtils.getTextFromResource(path = resourcePath).split("\n").drop(1) map { line =>
+        val tokens = line.split("\t")
+        (tokens(0), tokens(1))
+      }
+    }
+    print(targets)
```

Comment:
@sushmaakoju In general, let's remove debug print statements. 

The result of this print statement was:

```
[Lscala.Tuple2;@1984b1f
```

So I suspect you wanted to format the values as a string instead